internal/v2/repository/user/v1.go

```diff
@@ -0,0 +1,449 @@
+package user
```

Comment:
maybe it would make sense to create a separate package for these events e.g. user/v1/*.go